NEWS.md

```diff
@@ -1,3 +1,22 @@
+flux-accounting version 0.42.0 - 2025-02-11
+-------------------------------------------
+
+#### Fixes
+
+* python: use column names when accessing results of a query instead of row
+indices (#579)
+
+* doc: add man(1) page support, entries for DB administration subcommands
```

Comment:
Is this maybe not more of a new feature than a fix?

Reply:
Oh, duh. Good point - I'll change this to a feature. Thanks for catching this!